src/APIFunctions/User.js

```diff
@@ -11,11 +11,11 @@ let GENERAL_API_URL = process.env.REACT_APP_GENERAL_API_URL
  * @returns {UserApiResponse} Containing any error information or the array of
  * users.
  */
-export async function getAllUsers(token, query = null, page = null) {
+export async function getAllUsers(token, query = null, page = null, sortColumn = null, sortOrder = null) {
   let status = new UserApiResponse();
   await axios
     // get all user!
-    .post(GENERAL_API_URL + '/User/users', {
+    .post(GENERAL_API_URL + '/User/users' + `?sort=${sortColumn}&order=${sortOrder}`, {
```

Comment:
we can use the `URL` class, its build into js like

```js
const url = new URL(GENERAL_API_URL + '/User/users')

if (sortColumn) {
  url.searchParams.set('sort', sortColumn);
}

if (sortOrder) {
  url.searchParams.set('order', sortOrder);
}

await axios
    .post(url.href, {
// ...
```

Reply:
resolved with @syng828
[changed getAllUsers function into accepting object + use url class to…](https://github.com/SCE-Development/Clark/pull/1402/commits/2761592fda73f52b539f4f3796b2df6acac05fe6)